src/main/resources/english.yml

```diff
@@ -1,254 +1,259 @@
+config_and_lang_file_reloaded_successfully: 'Config and Language file reloaded successfully.'
+config_and_lang_file_could_not_be_loaded: 'Reload unsuccessful. See console for details.'
+status_yes: '&aYES'
+status_no: '&cNO'
+
+#Added in 0.03:
+msg_err_no_money: '&cThere is not enough money in the bank.'
+
+#Added in 0.04:
+msg_war_siege_peaceful_town_total_switches: '&b%d peaceful town%s %s peacefully switched nations.'
+msg_err_not_registered_1: '&c%s is not registered.'
+msg_error_must_be_num: '&bAmount must be a number.'
+
+#Added in 0.05:
+msg_err_not_being_sieged: '&bTown %s does not have any active sieges.'
+msg_err_town_not_registered: '&cTown %s is not registered.'
+
+#Scoreboard
+hud_title: 'Siege'
+hud_siegestatus: 'Siege Status'
+hud_attackers: 'ATK: '
+hud_defenders: 'DEF: '
+hud_points: 'Points: '
+hud_banner_control: 'Banner Ctrl: '
+hud_time_remaining: 'Time Left: '
+
+msg_err_peaceful_town_pvp_forced_off: '&cPeaceful towns have PVP forced OFF.'
+msg_swa_set_points_success: '&bSuccessfully set siege points to %d for %s.'
+msg_swa_resident_already_has_control: '&cResident %s already has control.'
+msg_swa_resident_doesnt_have_control: "&cResident %s doesn't have control."
+msg_swa_set_plunder_success: '&bSet plundered to %s for %s.'
+msg_swa_set_captured_success: '&bSet captured to %s for %s.'
+
+siege_status_in_progress: 'In Progress'
+siege_status_attacker_win: 'Attacker Win'
+siege_status_defender_win: 'Defender Win'
+siege_status_attacker_abandon: 'Attacker Abandon'
+siege_status_defender_surrender: 'Defender Surrender'
+siege_status_pending_attacker_abandon: 'Pending Attacker Abandon'
+siege_status_pending_defender_surrender: 'Pending Defender Surrender'
+siege_status_unknown: 'Unknown'
+
+dynmap_siege_title: 'Siege: %s'
+dynmap_siege_attacker: 'Attacker: %s'
+dynmap_siege_defender: 'Defender: %s'
+dynmap_siege_points: 'Points: %d'
+dynmap_siege_banner_control: 'Banner Control: %s'
+dynmap_siege_status: 'Status: %s'
+dynmap_siege_status_in_progress: 'In Progress'
+dynmap_siege_status_pending_surrender: 'Pending Surrender'
+dynmap_siege_status_pending_abandon: 'Pending Abandon'
+dynmap_siege_time_left: 'Time Left: %s'
+dynmap_siege_war_chest: 'War Chest: %s'
+
+#War sickness
+msg_you_received_war_sickness: '&cYou are not a participant of this siege and cannot be in the Siege Zone. Please leave to remove the war sickness. Common causes of war sickness: being in a peaceful town, not having a military rank or not being allied to either side.'
+msg_you_will_get_sickness: '&c&cYou are not a participant of this siege and cannot be in the Siege Zone. Please leave or you will receive war sickness in %s seconds.'
+
+#Added in 0.06:
+msg_inventory_degrade_warning: '&4WARNING: &cSome of your gear is close to breaking due to inventory degradation.'
+msg_swa_remove_siege_success: '&bSuccessfully removed siege.'
+msg_war_siege_occupied_towns_cannot_surrender: '&cOccupied towns cannot surrender.'
+
```

Comment:
:construction: 
This file is showing a lot of changes in the "github pr files" view, even though your PR only adds 2 fields. This is generally risky to review.  If at all possible, pls try to get ONLY these 2 fields showing up as the change. Perhaps grab a copy of the file from master, add the two fields, then replace the file in this branch.